src/WorkItems/VssConnectionWrapper.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.WebApi;
+
+namespace Microsoft.WorkItems
+{
+    internal class VssConnectionWrapper : IVssConnection
```

Comment:
>VssConnectionWrapper [](start = 19, length = 20)

I suggest "Facade" rather than "Wrapper" since that's the name of the design pattern for this technique.